lib/src/easy_date_time_formatting.dart

```diff
@@ -381,6 +396,25 @@ class EasyDateTimeFormatter {
 
     return null;
   }
+
+  /// Creates a new formatter by combining this formatter's pattern with [formatter2]'s pattern.
+  ///
+  /// The resulting formatter will format dates using the concatenated pattern.
+  /// This method uses the cache to return an existing formatter if one with the
+  /// combined pattern already exists.
+  EasyDateTimeFormatter combineFormatters(EasyDateTimeFormatter formatter2) {
```

Comment:
Just a thought - would a shorter name like combine() work here?
The current name is clear, but it can feel a bit verbose when used in chains:

```dart
// Current
dateFormatter.combineFormatters(timeFormatter)

// Alternative
dateFormatter.combine(timeFormatter)
```

Totally fine to keep it as-is if you prefer the explicitness 👍

Reply:
I think this is a good suggestion. done.